generators/app/tasks/appSetup/baseFilesTemplate.js

```diff
@@ -32,15 +33,24 @@ const FILES = [
   APP_NAVIGATOR,
   CUSTOM_TEXT,
   CUSTOM_TEXT_STYLES,
-  CONSTANTS,
-  REDUX_UTILS,
-  COLORS,
-  REDUX_UTILS_TESTS,
+  PLATFORM_CONSTANTS,
+  COLORS_CONSTANTS,
+  ARRAY_UTILS,
+  ARRAY_UTILS_TESTS,
```

Comment:
Repeating these definitions start to be messy. What about to import `FILES` and `TEMPLATE_FILES` directly from `'../../files'` ?

Reply:
Yeah, this is being annoying to use. I'll think of something different and implement it in a different pr.